app/[locale]/signin/page.tsx

```diff
@@ -1,3 +1,10 @@
+import { SignInForm } from '@/components/auth/signInForm';
+import { Card } from '@/components/ui/card';
+
 export default function SignIn() {
-  return <h1>Sign in</h1>;
+  return (
+    <Card>
+      <SignInForm />
+    </Card>
+  );
```

Comment:
```suggestion
  const text = useTranslations();
  return (
    <Card>
      <CardHeader>
        <CardTitle>{text(`login.header`)}</CardTitle>
        <CardDescription>{text(`login.header-description`)}</CardDescription>
      </CardHeader>
      <CardContent>
        <SignInForm />
      </CardContent>
    </Card>
  );
```

Я бы так сделала. А то получается, что у тебя половина компонента серверная, половина клиентская

Reply:
done